### PR TITLE
chore/gwe-89-toast-error

### PR DESCRIPTION
**Pull request by joshxfi**
- [x] #193 
- [x] Update: `next@latest`
- Fix(temp): push user to `/` on `/signup` when already registered
  - Before: shows error message (user exists) but moves user to `Step 3`
- [ ] Update `tailwindcss@latest` (encountered an error)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "framer-motion": "^4.1.17",
     "javascript-time-ago": "^2.3.10",
     "nanoid": "^3.1.29",
-    "next": "^11.1.2",
+    "next": "^12.0.7",
```
Comment (princejoogie) [inline package.json:32]: didnt we have problems with next v12?

**Review by princejoogie — CHANGES_REQUESTED** [review]
@joshxfi

File: package.json (hunk 1 above)
Comment (joshxfi) [reply package.json:32]: @princejoogie I no longer got a heap memory error from `12.0.7` this version was published 6 days ago

**Comment by joshxfi** [comment]
@princejoogie , I'm on the route where it used to have heap memory error (was using more than 2gb ram), currently on the same route w/ `v12.0.7` & it's using way more less. Also if I remember correctly `v11` was using around `1gb`

image

**Pull request merged**